packages/block-library/src/columns/editor.scss

```diff
@@ -89,13 +89,17 @@
 
 			// Beyond mobile, allow 2 columns.
 			@include break-small() {
-				flex-basis: 50%;
+				// Size should be half the available width, minus the margins added below
+				// In the editor, we also need to account for additional block padding
```

Comment:
Same here. I know this is really nitpicky, but we're basically a margin fix and some code comments polish from shipping this.